### DISCUSSION TITLE
...

### DISCUSSION THREAD

**Discussion by ghost** [jupyterlite/jupyterlite#1245] [Q&A]

**Comment by jtpio** [comment]
Thanks @terlan4 for reporting.

This seems more of a question to ask on the JupyterLab repository, as JupyterLite is based on JupyterLab.

Would you be able to open an issue or discussion on https://github.com/jupyterlab/jupyterlab?

Or even better create a new topic on the Jupyter Discourse? https://discourse.jupyter.org

Thanks!
